src/Tests/runTest.ts

```diff
@@ -43,11 +43,25 @@
 
 import {runTests} from '@vscode/test-electron';
 import * as path from 'path';
+import {argv} from 'process';
 
 async function main() {
+  let ci: boolean = false;
+  if (argv.length > 2) {
+    // front of two args
```

Comment:
To prevent confusion, maybe..?

```suggestion
    // front of two args are ignored
```